css/comp_lib_styles/components/card.css

```diff
@@ -0,0 +1,186 @@
+/* -------Styles for card component */
+
+.card {
+  border: 1px solid var(--clr-black-outline);
+  border-radius: var(--border-radius-md);
+  position: relative;
+}
+
+.card-vertical {
+  max-width: 12.5rem;
+  width: 100%;
+}
+
+/* -------Styles for card data - price, rating, title etc. */
+
+.card-data {
+  margin: 0.5rem 0.5rem 0 0.5rem;
+  position: relative;
+}
+
+.card-data > * + * {
+  margin-bottom: 0.5rem;
+}
+
+.card-img {
+  border-radius: 0.6rem 0.6rem 0 0;
+  display: block;
+}
+
+.card-vertical .btn {
+  width: 100%;
+}
+
+.card-title {
+  font-weight: var(--fw-medium);
+}
+
+/* --------styles for card badges, dismiss btn-------- */
+
+.card-badge,
+.card-dismiss {
+  position: absolute;
+  z-index: var(--z-index-three);
+  cursor: pointer;
+  max-width: 60%;
+}
+
+.card-dismiss {
+  right: -1px;
+  top: -1px;
+  padding: 0 0.5rem;
+  border-radius: 0 0.6rem 0 0.6rem;
+  background-color: var(--clr-black-disabled);
+}
+
+.card-badge {
+  top: -1px;
+  padding: 0.2rem 0.5rem;
+  font-weight: var(--fw-bold);
+  color: var(--clr-white-full);
+}
+
+.card-badge-top-right {
+  right: -1px;
+  border-radius: 0 0.6rem 0 0.6rem;
```

Comment:
This value for border-radius is being repeated so you can just create a variable for it.